apps/src/StudioApp.js

```diff
@@ -908,7 +918,13 @@ StudioApp.prototype.showInstructions_ = function(level, autoClose, showHints) {
     aniGifURL: level.aniGifURL,
     authoredHints: authoredHints
   });
-  React.render(instructionsContent, instructionsDiv);
+
+  // Create a div to eventually hold this content, and add it to the
+  // overall container. We don't want to render directly into the
+  // container just yet, because our React component could contain some
+  // elements that don't want to be rendered until they are in the DOM
+  var instructionsReactContainer = document.createElement('div');
+  instructionsDiv.appendChild(instructionsReactContainer);
```

Comment:
ordering, mostly; `instructionsDiv` contains this React element followed by the buttons that are generated on the next few lines. I figured creating and inserting it here, then "filling" it later was slightly clearer than creating it later and inserting it _before_ the buttons div.